src/azure-cli/azure/cli/command_modules/storage/_help.py

```diff
@@ -2078,10 +2127,6 @@
     text: |
         az storage metrics update --account-name MyAccount --api true --hour true --minute true --retention 10 --services bfqt
     crafted: true
-  - name: Update metrics settings for a storage account. (autogenerated)
```

Comment:
should not delete?

Reply:
We could reserve it. But every time Aladdin provide us a full set example, we just replace it and filter out those one duplicated with human written one

Reply:
reverted